mongo_orchestration/apps/sharded_clusters.py

```diff
@@ -27,6 +27,7 @@
     sharded_cluster_link, all_sharded_cluster_links, base_link,
     server_link, replica_set_link)
 from mongo_orchestration.common import *
+from mongo_orchestration.errors import RequestError
```

Comment:
Looks like this is missing from apps.replica_sets, too. Could you add it there as well?